### ISSUE TITLE
Ignore references to `__NSConcreteGlobalBlock` in `libsystem_blocks.dylib::__objc_classlist` and `libsystem_blocks.dylib::__objc_nlclslist`

### ISSUE THREAD

**Issue by WeiN76LQh** [droe/binja-blocks#17]
This is a minor issue but these errors occur when running the plugin against DSC with `libsystem_blocks.dylib` loaded:

```
[ScriptingProvider] Annotating global block 25eecd3b8
[ScriptingProvider] global block 25eecd3b8: AssertionError: 
[ScriptingProvider] Traceback (most recent call last):
[ScriptingProvider]   File "/Users/0xalsr/Library/Application Support/Binary Ninja/plugins/binja-blocks/blocks.py", line 686, in annotate_global_block_literal
[ScriptingProvider]     bl = BlockLiteral.from_data(bv, block_literal_data_var)
[ScriptingProvider]   File "/Users/0xalsr/Library/Application Support/Binary Ninja/plugins/binja-blocks/blocks.py", line 287, in from_data
[ScriptingProvider]     return cls(bv, is_stack_block, bl_data_var, isa, flags, reserved, invoke, descriptor)
[ScriptingProvider]   File "/Users/0xalsr/Library/Application Support/Binary Ninja/plugins/binja-blocks/blocks.py", line 367, in __init__
[ScriptingProvider]     assert (self.flags & BLOCK_IS_GLOBAL) != 0
[ScriptingProvider] AssertionError
[ScriptingProvider] Annotating global block 25eecd3e8
[ScriptingProvider] global block 25eecd3e8: AssertionError: 
[ScriptingProvider] Traceback (most recent call last):
[ScriptingProvider]   File "/Users/0xalsr/Library/Application Support/Binary Ninja/plugins/binja-blocks/blocks.py", line 686, in annotate_global_block_literal
[ScriptingProvider]     bl = BlockLiteral.from_data(bv, block_literal_data_var)
[ScriptingProvider]   File "/Users/0xalsr/Library/Application Support/Binary Ninja/plugins/binja-blocks/blocks.py", line 287, in from_data
[ScriptingProvider]     return cls(bv, is_stack_block, bl_data_var, isa, flags, reserved, invoke, descriptor)
[ScriptingProvider]   File "/Users/0xalsr/Library/Application Support/Binary Ninja/plugins/binja-blocks/blocks.py", line 367, in __init__
[ScriptingProvider]     assert (self.flags & BLOCK_IS_GLOBAL) != 0
[ScriptingProvider] AssertionError
```

This is purely because the symbol `__NSConcreteGlobalBlock` is referenced in a situation which has nothing to do with the defintion of a global block:

```c
libsystem_blocks.dylib::__objc_classlist (REGULAR) section started  {0x25eecd398-0x25eecd3c8}
25eecd398  struct objc_class_t* clsPtr___NSStackBlock__ = __NSConcreteStackBlock
25eecd3a0  struct objc_class_t* clsPtr___NSMallocBlock__ = __NSConcreteMallocBlock
25eecd3a8  struct objc_class_t* clsPtr___NSAutoBlock__ = __NSConcreteAutoBlock
25eecd3b0  struct objc_class_t* clsPtr___NSFinalizingBlock__ = __NSConcreteFinalizingBlock
25eecd3b8  struct objc_class_t* clsPtr___NSGlobalBlock__ = __NSConcreteGlobalBlock
25eecd3c0  struct objc_class_t* clsPtr___NSBlockVariable__ = __NSConcreteWeakBlockVariable
libsystem_blocks.dylib::__objc_classlist (REGULAR) section ended  {0x25eecd398-0x25eecd3c8}

libsystem_blocks.dylib::__objc_nlclslist (REGULAR) section started  {0x25eecd3c8-0x25eecd3f8}
25eecd3c8  struct objc_class_t* clsPtr___NSStackBlock__ = __NSConcreteStackBlock
25eecd3d0  struct objc_class_t* clsPtr___NSMallocBlock__ = __NSConcreteMallocBlock
25eecd3d8  struct objc_class_t* clsPtr___NSAutoBlock__ = __NSConcreteAutoBlock
25eecd3e0  struct objc_class_t* clsPtr___NSFinalizingBlock__ = __NSConcreteFinalizingBlock
25eecd3e8  struct objc_class_t* clsPtr___NSGlobalBlock__ = __NSConcreteGlobalBlock
25eecd3f0  struct objc_class_t* clsPtr___NSBlockVariable__ = __NSConcreteWeakBlockVariable
libsystem_blocks.dylib::__objc_nlclslist (REGULAR) section ended  {0x25eecd3c8-0x25eecd3f8}
```

Its not a big deal as I don't believe it causes any problems other than those errors to occur.

**Comment by droe** [comment]
Yeah this is purely a UX issue.  Currently testing a change to ignore those two sections as well as less noisy logging of harmless "this does not look like a block" type errors that are unlikely to indicate a plugin issue.

**Comment by droe** [comment]
Addressed in bd56a84.
